config/prod/rstudio_ami_ec2.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2.yaml
+stack_name: rstudio-ami-instance-ec2
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "Infrastructure"
+  # The resource owner
+  OwnerEmail: "jineta.banerjee@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+#  VpcName: "{{stack_group_config.default_vpc}}"
+#  KeyName: "{{stack_group_config.default_key_pair}}"
+#  AMIId: "{{stack_group_config.default_awslinux2_ami}}"
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "256"
+  # (Optional) Base instance on AMI (default: ami-0de53d8956e8dcf80)
+  AMIId: "ami-01d45add662773bed"
```

Comment:
where is this AMI from? can you add a comment with this info or a link to some documentation?

Reply:
The AMI is an RStudio AMI from http://www.louisaslett.com/RStudio_AMI/

Reply:
the AMI you chose is in US-West-2, we only support US-EAST-1 region.  Also it's not the latest version of the AMI.  I think the one you want is `ami-0226a8af83fcecb43`?  that's the latest version of US-EAST-1 region.

Reply:
Actually, I want the Studio with R version 3.5 (which is the earlier version), since synapser is not yet compatible with R 3.6 (latest) version. Will remake the PR to reflect the US-East region AMI.